SPECS/kernel-mshv/kernel-mshv.spec

```diff
@@ -1,7 +1,7 @@
 
 %global security_hardening none
 %global sha512hmac bash %{_sourcedir}/sha512hmac-openssl.sh
-%define uname_r %{version}-hvl1.m2
+%define uname_r %{version}-2.cm2
```

Comment:
nit: Can this be %{version}-%{release}  ?

Reply:
Ahh. I got this error, so I hardcoded the `-2.cm2` at the end
<img width="849" alt="image" src="https://user-images.githubusercontent.com/30801614/227613269-89529b51-d3db-4463-ab02-0ca464c53198.png">
But the 2 probably comes from release. So I think it could be %{version}-%{release}.cm2

Reply:
Forgot to update this. Cameron is right, %{version}-%{release}  works and the `.cm2` comes from [dist](https://github.com/microsoft/CBL-Mariner/pull/5146/files#diff-e030ce8ff9f0cb3e493e5851321344b2377eb88823ea616c08b55fe6f3f56955R14)

Reply:
Got it. Just updated.